invoice.js

```diff
@@ -0,0 +1,160 @@
+const config = require('./config');
```

Comment:
how is this script run?

Reply:
Manually. We could probably set up a cron job, but I'd rather supervise it.